src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setAsDone() {
+        isDone = true;
+    }
+
+    public void setAsUndone() {
+        isDone = false;
+    }
+
+	public String getTaskIcon() {
+		return " ";
+	}
+}
```

Comment:
Seems to be only used once in your main and quite trivial, perhaps could be removed or replaced as a constant in main